client/components/thank-you-v2/index.tsx

```diff
@@ -29,9 +32,18 @@ export default function ThankYouV2( props: ThankYouV2Props ) {
 		showSuccessAnimation = true,
 	} = props;
 
+	const [ shouldShowAnimation, setShouldShowAnimation ] = useState( false );
+
+	useEffect( () => {
+		if ( showSuccessAnimation ) {
+			setShouldShowAnimation( true );
+			setTimeout( () => setShouldShowAnimation( false ), CONFETTI_ANIMATION_DELAY_MS );
```

Comment:
Do we really need to toggle the value back to `false`..? It seems sufficient to introduce a state hook to ensure we don't flip back and forth. 

If we do decide to keep this `setTimeout` call, we should return a cleanup function from this `useEffect` hook to clear the timeout. I think it's better to just remove this line, though.

Reply:
That makes sense, we never need to toggle the value back to `false`
I have removed the line